src/contact.html

```diff
@@ -7,20 +7,32 @@
     <h1 class="title-heading">Contact Us</h2>
 	</div>
 </section>
-<section>
-	<h2> Get in touch </h2>
+<section class="col-lg-3 col-md-7 col-sm-9 col-xs-12 position central">
+	<h2>Get in touch</h2>
 	<form method="POST" action="//formspree.io/help@womenhackfornonprofits.com">
-    <label for="name">Name:</label>
-		<input type="text" name="name" placeholder="E.g. Jon Snow" required>
-    <label for="email">Email Address:</label>
-		<input type="email" name="_replyto" placeholder="E.g. jon.snow@example.com" required>
-		<input type="hidden" name="_subject" value="New Submission: WHFN Website Contact Form"/>
-		<input type="text" name="_gotcha" style="display:none"/>
-    <label for="tel">Phone Number:</label>
-		<input type="tel" name="phone" placeholder="E.g. 07912 345 678">
-    <label for="message">Enter your message here:</label>
-		<textarea name="message" rows="5" placeholder="E.g. Hello, just wanted to tell you I think you are awesome" required></textarea>
-		<input type="submit" value="Send">
+		<div class="form-group">
+	    	<label for="name">Name</label>
```

Comment:
This is not specifically for this line, but generally for throughout this PR (wanted to use this line as an example though) 

- Wrap radio and checkbox inputs and their text in labels. No need for `for` attributes. The wrapping automatically associates the two.

```
<!-- bad -->
<label for="email">
<input id="email" name="email">

<!-- good -->
<label>Name: <input name="email"></label>
```